meta-resin-supervisor/recipes-connectivity/avahi/avahi/no-gobject-introspection.patch

```diff
@@ -0,0 +1,65 @@
+Deactivate completely GOBJECT INTROSPECTION.
```

Comment:
It would be great to have a comment here on why we're deactivating this.

Reply:
@pcarranzav Done!